trunk/Admin/js/admin-quickbar-admin.js

```diff
@@ -125,12 +125,12 @@
  * Set localStorage
  */
 function setPostListStorage() {
-  let plStorage = {};
+  let postlistStorage = {};
```

Comment:
Should be named `postListStorage` like the method name.

Reply:
Done